internal/storage/elasticsearch/config/config_test.go

```diff
@@ -1470,6 +1472,61 @@ func TestBulkCallbackInvoke_NilResponse(t *testing.T) {
 	)
 }
 
+func TestAuthExtensionConfig(t *testing.T) {
```

Comment:
I don't see the need for this test. First, wantErr is always false, but more importantly what is it actually validating? This package has no knowledge of "extensions".